pkgs/applications/video/dvdbackup/default.nix

```diff
@@ -11,6 +11,12 @@ stdenv.mkDerivation rec {
 
   buildInputs = [ libdvdread libdvdcss dvdauthor ];
 
+  patchFlags = [ "-p0" ];
+  patches = [ (fetchpatch {
```

Comment:
do you mind adding a comment as to why this is necessary and when it could be removed? (if it's already in master)

Reply:
Added a reference to the related upstream bug report where the patch seems to be originated.